index.html

```diff
@@ -2,11 +2,28 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>TIY Houston</title>
+    <title>Houston</title>
 
     <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-
+    <div class="container">
+      <img class="banner" src="img/dogs.jpg" alt="">
+      <img class="image-logo" src="img/mal.jpg" alt="">
+    </div>
+    <div class="row">
+      <div class="main">
+        <div class="col-left">
+          <h6>The Iron Yard |</h6>
+          <h6>Houston</h6>
+          <h6 class="you-will-never-get-this-la-la-la-la-la">Happenings and Updates from The Iron Yard in Houston, TX</h6>
+        </div>
+        <div class="col-right">
+          <h2>September 22 Starts a New Class of The Iron Yard Houston Students</h2>
+          <p>by Someone_realSpecialLike over at <a class="shittacca" href="theironyard.com/nashville">placecage</a></p>
+          <img class="image"src="http://placecage.com/400/200" alt="">
```

Comment:
Can you add some extra paragraph content to the bottom of this to make a bit more on the page?